docs/app/Examples/collections/Table/States/TableWarningShorthand.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Table } from 'stardust'
+
+const headerRow = [
+  'Name',
```

Comment:
+1 for the rename of `bodyRow` => `renderBodyRow` to differentiate it more. Not completely opposed to making the other two functions in the future, but for now I'd prefer keeping it as-is. Perhaps when we add column/cell shorthands it will give a little more clarity.